src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+  .calendar {
+    flex-wrap: wrap;
+    gap: $calendar-gap;
+    width: calc(7 * $calendar-day-width + 7px);
+    list-style: none;
+    display: flex;
+
+    &__day {
+      cursor: pointer;
+      background-color: $calendar-day-color;
+      border: 1px solid black;
+      justify-content: center;
+      align-items: center;
+      height: $calendar-day-height;
+      width: $calendar-day-width;
+      transition: 500ms;
+      display: flex;
+      box-sizing: border-box;
+
+      &:hover {
+        background-color: #FFBFCB;
+        transform: translateY(-20px);
+
+      }
+
+      &::before {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: $calendar-day-font-size;
+        content: "";
```

Comment:
The '::before' pseudo-element should not have an empty 'content' property. If the intention is to dynamically set the content using SCSS, then this line can be removed as the content is set within the @for loop.